RegenAllByteCodeNoBuild.cmd

```diff
@@ -0,0 +1,62 @@
+::-------------------------------------------------------------------------------------------------------

```

Comment:
Wouldn't it be better if `RegenAllByteCode.cmd` called this script after building everything needed ?

Also, I was thinking we probably should move these scripts to the `tools` folder instead of leaving them at the root. (Would require a fix of the regen script in chakra full). If you don't want to do this now, we can create an issue and I'll clean it up later.